menuv.lua

```diff
@@ -184,6 +184,40 @@ function MenuV:CreateMenu(title, subtitle, position, r, g, b, size, texture, dic
     return self.Menus[index] or menu
 end
 
+--- Create a menu that inherits properties from another menu
+---@param parent Menu|string Menu or UUID of menu
+---@param overrides table<string, string|number> Properties to override in menu object (ignore parent)
+function MenuV:CreateChildMenu(parent, overrides)
+    overrides = Utilities:Ensure(overrides, {})
+
+    local uuid = Utilities:Typeof(parent) == 'Menu' and parent.UUID or Utilities:Typeof(parent) == 'string' and parent
+
+    if (uuid == nil) then return end
+
+    parentMenu = self:GetMenu(uuid)
```

Comment:
it should be: (it's now a global variable)
```lua
local parentMenu = self:GetMenu(uuid)
```

Reply:
Oh sorry, I didn't really notice that 😅